### DISCUSSION TITLE
`copilot.vim` plugin crashing on start

### DISCUSSION THREAD

**Discussion by alichtman** [community/community#58745] [Copilot Conversations]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

`Copilot.vim` just started crashing on load for me. I don't know why, and it might not be a copilot issue (but instead an nvim breaking change or something). I installed it with vim-plug:

```vim
Plug 'github/copilot.vim'
```

```bash
$ vim --version
NVIM v0.10.0-dev
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Compilation: /usr/bin/cc -O2 -g -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wvla -Wdouble-promotion -Wmissing-noreturn -Wmissing-format-attribute -Wmissing-prototypes -fno-common -Wno-unused-result -Wimplicit-fallthrough -fdiagnostics-color=auto -fstack-protector-strong -DUNIT_TESTING -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_TS_HAS_SET_MAX_START_DEPTH -I/usr/include/luajit-2.1 -I/usr/include -I/build/neovim-7GHzWb/neovim-0.10.0~ubuntu1+git202306212018-e27377e33-9b397864c/.deps/usr/include -I/build/neovim-7GHzWb/neovim-0.10.0~ubuntu1+git202306212018-e27377e33-9b397864c/build/src/nvim/auto -I/build/neovim-7GHzWb/neovim-0.10.0~ubuntu1+git202306212018-e27377e33-9b397864c/build/include -I/build/neovim-7GHzWb/neovim-0.10.0~ubuntu1+git202306212018-e27377e33-9b397864c/build/cmake.config -I/build/neovim-7GHzWb/neovim-0.10.0~ubuntu1+git202306212018-e27377e33-9b397864c/src

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info
```

```
Error detected while processing function <lambda>4[1]..<SNR>53_Start[4]..copilot#agent#New:
line   25:
E5108: Error executing lua /usr/share/nvim/runtime/lua/vim/version.lua:129: attempt to concatenate field 'build' (a userdata value)
stack traceback:
        /usr/share/nvim/runtime/lua/vim/version.lua:129: in function 'tostring'
        /usr/share/nvim/runtime/lua/vim/lsp.lua:1381: in function 'initialize'
        /usr/share/nvim/runtime/lua/vim/lsp.lua:1726: in function 'start_client'
        ...n/.local/share/nvim/plugged/copilot.vim/lua/_copilot.lua:14: in function <...n/.local/share/nvim/plugged/copilot.vim/lua/_copilot.lua:3>
```

`:Copilot version` fails with the same error, but I'm pretty sure it's 1.8.4.

```
copilot.vim on  release via 🌙 took 52ms
03:51:54 AM ➜ pwd
/home/alichtman/.local/share/nvim/plugged/copilot.vim

copilot.vim on  release via 🌙 took 1ms
03:52:07 AM ➜ git show --shortstat
commit 1358e8e45ecedc53daf971924a0541ddf6224faf (HEAD -> release, origin/release, origin/HEAD)
Author: Tim Pope <tpope@github.com>
Date:   Fri Apr 28 17:28:11 2023 -0400

    Copilot.vim 1.8.4

 8 files changed, 38 insertions(+), 26 deletions(-)
```

Any idea what's up here?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @alichtman ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by alichtman** [reply]
Hey, @queenofcorgis. Thanks for the response!

Just to clarify: I'm reporting a bug in a Github managed repo (https://github.com/github/copilot.vim) that has the issues category disabled. Are you sure that Discussions is the right category for this?

Edit: Oh! I see what got changed. Thanks

**Comment by tpope** [comment]
Hi @alichtman, looks like this was a Neovim bug: https://github.com/neovim/neovim/commit/43e76cc3462bc5bcf2b6ade8af1c36e21d3da3c9. Upgrading your Neovim should fix it.
